Docs/Api/LibMongocryptDocs.shfbproj

```diff
@@ -0,0 +1,43 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file is generated via `sandcastle` GUI`. But then I've updated this file a little bit to follow to the structure of `CSharpDriverDocs.shfbproj`

Reply:
Why is there a new `.shfbproj` file?

Can we just ignore `MongoDB.LibMongocrypt` entirely?

We don't really want that wrapper DLL in the API docs anyway.

Reply:
>Can we just ignore MongoDB.LibMongocrypt entirely?
I haven't find a way for it yet